bridges/snowbridge/pallets/system/src/lib.rs

```diff
@@ -564,13 +568,15 @@ pub mod pallet {
 				create_asset_xcm,
 				transfer_asset_xcm,
 				register_token,
+				destination_max_transfer_asset_xcm,
 			};
 			Self::send(PRIMARY_GOVERNANCE_CHANNEL, command, PaysFee::<T>::No)?;
 
 			Self::deposit_event(Event::<T>::SetTokenTransferFees {
 				create_asset_xcm,
 				transfer_asset_xcm,
 				register_token,
+				destination_max_transfer_asset_xcm,
```

Comment:
Should it not also be checked that it is > 0 in line `563`? (https://github.com/Snowfork/polkadot-sdk/pull/133/files#diff-4c470a9af5f416d98631522dc9be3d607ad099b30a980747ac74b6877f17b003R563)